R/tinyplot.R

```diff
@@ -782,40 +759,31 @@ tinyplot.default = function(
   facet_attr = attributes(facet) ## TODO: better solution for restoring facet attributes?
   null_facet = is.null(facet)
 
+  tmp = sanitize_xylab(
+    x = x, xlab = xlab, x_dep = x_dep, xmin_dep = xmin_dep, xmax_dep = xmax_dep,
+    y = y, ylab = ylab, y_dep = y_dep, ymin_dep = ymin_dep, ymax_dep = ymax_dep,
+    type = type)
+  list2env(tmp, environment())
```

Comment:
Same comment as above; I think it's safer to do this directly than creating a temporary variable in case something goes wrong and the value of `tmp` doesn't get (re)assigned properly.